client/src/dashboard/pages/contacts/Contacts.jsx

```diff
@@ -37,6 +40,8 @@ function Contacts() {
     { title: 'Extension', field: 'exten' },
   ];
 
+  const filteredContacts = filterSearch(contacts, searchTerm, [contacts.first_name, contacts.last_name]);
```

Comment:
Any reason why we need first_name and last_name as additional values?

Reply:
if you can only remember the first or last name of someone you want to find you can still find them

Reply:
Is it already included in a Contact object though?

Reply:
Removed extraneous data pass